utils.bash

```diff
@@ -0,0 +1,3 @@
+get_pods() {
```

Comment:
It would be a complete overkill. 🔨 
We can always convert in the future if needed.